lib/matplotlib/colors.py

```diff
@@ -960,6 +960,229 @@ def scaled(self):
         return (self.vmin is not None and self.vmax is not None)
 
 
+class FuncNorm(Normalize):
+    """
+    Creates a normalizer using a custom function
+
+    The normalizer will be a function mapping the data values into colormap
```

Comment:
@efiring  Thanks for pointing that out, I always thought the init could either be documented in the class, on in the init itself. I will do it the way you suggested.